tests/rust-integration-tests/integration_test/src/tests/seccomp_notify/seccomp_agent.rs

```diff
@@ -24,14 +31,14 @@ pub fn recv_seccomp_listener(seccomp_listener: &Path) -> SeccompAgentResult {
         None,
     )
     .context("failed to create seccomp listener socket")?;
-    socket::bind(socket, &addr).context("failed to bind to seccomp listener socket")?;
+    let socket = socket;
```

Comment:
Why is this assignment `let socket = socket;` necessary? Something left from previous iterations?

Reply:
My mistake. Initially, there was `let socket = std::mem::ManuallyDrop::new(socket);`, and later, I removed `ManuallyDrop::new`. 
I will fix here. Thanks for your review!